include/mbedtls/gcm.h

```diff
@@ -375,5 +377,6 @@ int mbedtls_gcm_self_test( int verbose );
 }
 #endif
 
+#endif /* MBEDTLS_GCM_C */
```

Comment:
Ahh, spent an hour trying to figure out why my C++ code does not compile and throws mysterious errors :smile: Finally, this is the reason.. this line should be in line 375.

Else we have:

``
extern "C" {
``

that is not closed when MBEDTLS_GCM_C is not defined.

Reply:
Should be done :)

Reply:
Should be fixed with a force-push